src/App.js

```diff
@@ -55,11 +68,7 @@ const ProtectedRoute = ({ backend, children, path }) => {
         setBadDomain(backend.badDomain);
       }
     });
-  }, [backend]);
-
-  useEffect(() => {
-    checkVerification();
-  }, [checkVerification]);
+  }, [backend, loggedIn, authLoaded, history]);
```

Comment:
Question: is there any concern of this running multiple times if/when these four values change? Or are we safe because of `if (!loggedIn && !authLoaded) {` and `if (!loggedIn && authLoaded) {`

Reply:
Actually I just refactored this so that `authLoaded` and `loggedIn` aren't getting set separately. This causes 1 less render inside `useEffect`. This works better and seems safer:

```
{
    isInitializing: <boolean>, // default true
    isLoggedIn: <boolean>,
    user: <user>,
}
```